webclient/src/hooks/useDebounce.ts

```diff
@@ -0,0 +1,13 @@
+import { useCallback } from 'react';
+import { debounce, DebouncedFunc } from 'lodash';
+
+type UseDebounceType = (...args: any) => any;
+const DEBOUNCE_DELAY = 250;
```

Comment:
should have time increment, DEBOUNCE_DELAY_SEC or smth

Reply:
Milliseconds is kinda the default standard.

Reply:
It's dangerous to assume unit defaults